src/app-bar.jsx

```diff
@@ -1,4 +1,5 @@
 import React from 'react';
+import ReactDOM from 'react-dom';
```

Comment:
wanted to ask the same thing, but looking at the code, it's very tightly coupled with the code. I don't know how it can be decoupled.